pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,58 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OLMConfigSpec is the spec for an OLMConfig resource.
+type OLMConfigSpec struct {
+	Features Features `json:"features,omitempty"`
+}
+
+// Features contains the list of configurable OLM features.
+type Features struct {
+	// +kubebuilder:default=false
+
+	// DisableCopiedCSVs is used to disable OLM's "Copied CSV" feature
+	// for operators installed at the cluster scope, where a cluster
+	// scoped operator is one that has been installed in an
+	// OperatorGroup that targets all namespaces.
+	// When reenabled, OLM will recreate the "Copied CSVs" for each
+	// cluster scoped operator.
+	DisableCopiedCSVs bool `json:"disableCopiedCSVs,omitempty"`
+}
+
+// OLMConfigStatus is the status for an OLMConfig resource.
+type OLMConfigStatus struct {
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:storageversion
+// +kubebuilder:resource:categories=olm,scope=Cluster
```

Comment:
Should this resource be cluster scoped?

Reply:
Based on the original [enhancement](https://github.com/operator-framework/enhancements/pull/92/files#diff-c21d583efb087d4f578e5a734ef0e50615eec9743cb5ffa9fb388f32041bc6b9R105) it should be.